src/domain/app/App.tsx

```diff
@@ -69,6 +55,27 @@ const ApolloWrapper: React.FC<PropsWithChildren> = ({ children }) => {
 };
 
 const App: React.FC = () => {
+  const matomoTracker = useMemo(
+    () =>
+      new MatomoTracker({
+        urlBase: getValue(import.meta.env.REACT_APP_MATOMO_URL_BASE, ''),
+        siteId: import.meta.env.REACT_APP_MATOMO_SITE_ID,
+        srcUrl: import.meta.env.REACT_APP_MATOMO_SRC_URL,
+        enabled: import.meta.env.REACT_APP_MATOMO_ENABLED === 'true',
+        configurations: {
+          setDocumentTitle: `${document.location}${document.title}`,
```

Comment:
In the end I did not set setDocumentTitle at all, I think the default value looks fine.